.github/ISSUE_TEMPLATE/git-failure-due-to-authentication.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Git failure due to authentication
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Which Version of GCM are you using ?**
+You can see the GCM version by setting the env variable GCM_TRACE=1 and performing a command.
+<!-- Ex: 1.18.1, 1.18.2, etc.. -->
+
+**Which service are you trying to connect to**
+* Azure Devops
+    * [ ] Hosted
+    * [ ] On-Prem
```

Comment:
Let's do this as:
```
[ ] Azure DevOps
[ ] Azure DevOps Server (TFS/on-prem)
```
Rather than nested under Azure DevOps. They're quite different from the auth perspective.